app/publishing/publish-statement.js

```diff
@@ -21,16 +21,21 @@ const publishStatement = async (request) => {
     throw new Error('Could not check for duplicates')
   }
 
+  const publishStatementType = request?.emailTemplate ? EMAIL : LETTER
```

Comment:
TODO: we need to discuss what request structure will determine the difference between email and letter.